jp.yaml

```diff
@@ -201,7 +201,7 @@ months:
     wday: 1
     week: 2
     year_ranges:
-      from: 2021
+      from: 2022
```

Comment:
In both the years 2020 and 2021, the sports days were changed. Therefore, Sports Day in October was defined to begin in 2022.

Reply:
Whoops,　I have missed it......
Thank you for checking.:)